src/engine-actions/get-channels-for-address.js

```diff
@@ -0,0 +1,34 @@
+const { networkAddressFormatter } = require('../utils')
+const { listChannels, listPendingChannels } = require('../lnd-actions')
+
+/**
+ * Get all channels to the given address
+ * @param {string} address
+ * @returns {Array} channels
+ */
+async function getChannelsForAddress (address) {
+  const [
+    { channels = [] },
+    { pendingOpenChannels = [] }
+  ] = await Promise.all([
+    // The response from listChannels consists of channels that may be active or inactive
+    listChannels({ client: this.client }),
+    listPendingChannels({ client: this.client })
+  ])
+
+  const normalizedPendingChannels = pendingOpenChannels.map(chan => chan.channel)
+
+  if (channels.length === 0 && normalizedPendingChannels.length === 0) {
+    this.logger.debug('getChannelsForAddress: No channels exist')
```

Comment:
@martineehrlich should we return here instead of going through the rest of the code? (not sure if we do this in the other files)

Reply:
Looks like we do return early for `getMaxChannel`. We should return early here with `return {}`